kahuna/public/stylesheets/main.css

```diff
@@ -308,9 +308,24 @@ a:focus {
     display: inline-block;
 }
 
-/* ==========================================================================
-   Generics
-   ========================================================================== */
+.side-padded {
+    padding: 0 10px;
+}
+
+.clickable:hover {
```

Comment:
Only gr-confirm-delete at the moment (does padding on at bottom of crops and on upload page). The 'side-padded' rule is on components that might be used somewhere other than topbar in the future?